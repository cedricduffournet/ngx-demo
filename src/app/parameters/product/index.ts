export * from './parameter-product.component';
