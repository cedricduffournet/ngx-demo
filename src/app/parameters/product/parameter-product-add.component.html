<app-product-add-view></app-product-add-view>
