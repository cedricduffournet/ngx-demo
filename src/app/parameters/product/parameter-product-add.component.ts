import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';

@Component({
  selector: 'app-parameter-product-add',
  templateUrl: './parameter-product-add.component.html',
  changeDetection: ChangeDetectionStrategy.OnPush
})
export class ParameterProductAddComponent implements OnInit {
  constructor() {}

  ngOnInit() {}
}
