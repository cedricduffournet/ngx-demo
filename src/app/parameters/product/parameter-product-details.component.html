<app-product-details-view></app-product-details-view>
