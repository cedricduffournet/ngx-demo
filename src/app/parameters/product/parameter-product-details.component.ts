import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';

@Component({
  selector: 'app-parameter-product-details',
  templateUrl: './parameter-product-details.component.html',
  changeDetection: ChangeDetectionStrategy.OnPush
})
export class ParameterProductDetailsComponent implements OnInit {
  constructor() {}

  ngOnInit() {}
}
