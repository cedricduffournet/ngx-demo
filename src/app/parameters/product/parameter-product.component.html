<app-product-list-view></app-product-list-view>
