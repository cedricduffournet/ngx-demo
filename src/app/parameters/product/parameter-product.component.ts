import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';

@Component({
  selector: 'app-parameter-product',
  templateUrl: './parameter-product.component.html',
  changeDetection: ChangeDetectionStrategy.OnPush
})
export class ParameterProductComponent implements OnInit {
  constructor() {}

  ngOnInit() {}
}
