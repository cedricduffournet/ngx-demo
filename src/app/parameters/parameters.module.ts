import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { ReactiveFormsModule } from '@angular/forms';
import { TranslateModule } from '@ngx-translate/core';

import { PageInnerModule } from '@app/shared/page-inner/page-inner.module';
import { CivilityModule } from '@app/civility/civility.module';
import { ProductCategoryModule } from '@app/product-category/product-category.module';
import { ProductModule } from '@app/product/product.module';

import { ListParametersComponent } from '@app/parameters/containers';
import { ReferencedTableMenuComponent } from '@app/parameters/components';
import { ParameterCivilityComponent } from '@app/parameters/civility';
import { ParameterProductCategoryComponent } from '@app/parameters/product-category';
import { ParameterProductComponent, ParameterProductDetailsComponent, ParameterProductAddComponent } from '@app/parameters/product';

import { routing } from './parameters.routing';

@NgModule({
  imports: [
    CommonModule,
    TranslateModule,
    PageInnerModule,
    CivilityModule,
    routing,
    ReactiveFormsModule,
    ProductCategoryModule,
    ProductModule
  ],
  declarations: [
    ListParametersComponent,
    ReferencedTableMenuComponent,
    ParameterCivilityComponent,
    ParameterProductCategoryComponent,
    ParameterProductComponent,
    ParameterProductDetailsComponent,
    ParameterProductAddComponent
  ]
})
export class ParametersModule {}
