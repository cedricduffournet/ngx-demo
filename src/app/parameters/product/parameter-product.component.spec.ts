import { Component } from '@angular/core';
import { TestBed, ComponentFixture } from '@angular/core/testing';
import { ParameterProductComponent } from '@app/parameters/product';

@Component({ selector: 'app-product-list-view', template: '' })
class ListProductViewStubComponent {}

describe('ParameterProductComponent', () => {
  let fixture: ComponentFixture<ParameterProductComponent>;
  let component: ParameterProductComponent;

  beforeEach(() => {
    TestBed.configureTestingModule({
      declarations: [ParameterProductComponent, ListProductViewStubComponent]
    });

    fixture = TestBed.createComponent(ParameterProductComponent);
    component = fixture.componentInstance;
  });

  it('should be created', () => {
    fixture.detectChanges();

    expect(fixture).toMatchSnapshot();
  });
});
