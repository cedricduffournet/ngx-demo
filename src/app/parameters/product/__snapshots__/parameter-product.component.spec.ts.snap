// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ParameterProductComponent should be created 1`] = `
<app-parameter-product>
  <app-product-list-view />
</app-parameter-product>
`;
